docs/options.md

```diff
@@ -327,6 +327,32 @@ Platform-specific variants also available:<br/>
 CIBW_TEST_EXTRAS: test,qt
 ```
 
+### `CIBW_BEFORE_TEST` {: #before-test}
+> Execute a shell command preparing test environment
+
+A shell command to run after setup test environment. This option allows you to run a command in **each** Python environment before the `pip wheel` command. This is useful if you need to install non pip package, change values of environment variables
+or perform multi step pip installation (like install `scikit-build` or `cython` before install test package)
+
+If dependencies are required to build your wheel (for example if you include a header from a Python module), set this to `pip install .`, and the dependencies will be installed automatically by pip. However, this means your package will be built twice - if your package takes a long time to build, you might wish to manually list the dependencies here instead.
+
+The active Python binary can be accessed using `python`, and pip with `pip`; `cibuildwheel` makes sure the right version of Python and pip will be executed. `{project}` can be used as a placeholder for the absolute path to the project's root and will be replaced by `cibuildwheel`.
+
+On Linux and macOS, the command is run in a shell, so you can write things like `cmd1 && cmd2`.
+
+Platform-specific variants also available:<br/>
+ `CIBW_BEFORE_TEST_MACOS` | `CIBW_BEFORE_TEST_WINDOWS` | `CIBW_BEFORE_TEST_LINUX`
+
+#### Examples
+```yaml
+# install packages needed to build test dependencies
+CIBW_BEFORE_TEST: pip install cmake scikit-build
```

Comment:
See above, I'm not sure why these aren't TEST_REQUIRES

Reply:
As I mentioned before. Some dependencies need to be importable in `setup.py`. And still some authors of packages does not provide wheels or it takes over month since release of python 3.8 to wheel for this python. And such things allow maintainer to have full test case without waiting on other persons. 

```
File "/tmp/pip-install-f9rmc9hk/cmake/setup.py", line 7, in <module>
          from skbuild import setup
      ModuleNotFoundError: No module named 'skbuild'
```

Reply:
@joerick I know `setup_requires` doesn't always work, for some reason. So that would be good.

But indeed, this might not be the perfect example to put in the docs, either?